packages/components/src/range-control/styles/range-control-styles.ts

```diff
@@ -290,13 +291,31 @@ export const Tooltip = styled.span< TooltipProps >`
 	) }
 `;
 
+const inputNumberWidth = ( { size }: InputNumberProps ) => {
+	const sizes = {
+		default: space( 16 ),
+		'__unstable-large': space( 20 ),
+	};
+
+	if ( size === '__unstable-large' ) {
+		return css( {
+			width: `${ sizes[ '__unstable-large' ] } !important`,
+		} );
+	}
+
+	return css`
+		width: ${ sizes.default } !important;
+	`;
+};
+
 // @todo: Refactor RangeControl with latest HStack configuration
 // @wordpress/components/ui/hstack.
 export const InputNumber = styled( NumberControl )`
 	display: inline-block;
 	font-size: 13px;
+	margin-bottom: 0 !important;
```

Comment:
Why is this necessary? (Both the `margin-bottom` and the `!important`)

Reply:
I'm guessing you're referring to the margin-bottom [added](https://github.com/WordPress/gutenberg/blob/584c1d27a92db85491fe4bf3eb883d689faa3f5f/packages/block-editor/src/components/block-inspector/style.scss#L13-L19) by the `block-inspector` stylesheet? This is a `block-editor` package concern, so it should be dealt with in that package rather than in `@wordpress/components`.

Can it be addressed similarly to the FocalPointPicker here?

https://github.com/WordPress/gutenberg/blob/584c1d27a92db85491fe4bf3eb883d689faa3f5f/packages/block-editor/src/components/block-inspector/style.scss#L21-L25

Reply:
This worked well, thanks for the suggestion @mirka